apps/arweave/src/ar_http_iface_middleware.erl

```diff
@@ -2007,6 +2007,15 @@ handle_get_chunk(OffsetBinary, Req, Encoding) ->
 							{{true, Packing}, _StoreID} when RequestedPacking == any ->
 								ok = ar_semaphore:acquire(get_chunk, infinity),
 								{Packing, ok};
+							{true, _StoreID} ->
```

Comment:
I think this missing clause was addressed in master. I suggest we go with the master version for now (since it's undergone some testing already)